classes/author.rb

```diff
@@ -0,0 +1,15 @@
+class Author
+  attr_accessor :items, :last_name, :first_name
```

Comment:
it would be better if you changed the items array attribute to the reader as represented in the diagram, though you will be able to update it using the add_item method.